modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/custom_fields.js

```diff
@@ -1270,6 +1270,30 @@ AUI.add(
 
 					var keysPressed = {};
 
+					var onKeyDown = function (domEvent) {
+						if (domEvent.keyCode === 16) {
+							keysPressed[domEvent.keyCode] = true;
+						}
+					};
+
+					var onKeyUp = function (domEvent) {
+						if (domEvent.keyCode === 16) {
+							delete keysPressed[domEvent.keyCode];
+						}
+					};
+
+					var closePopoverOnKeyboardNavigation = function (instance) {
+						instance.hide();
+
+						keysPressed = {};
+
+						var trigger = A.one('#<%= nameId %>');
```

Comment:
This won't work, `<%= %>` only works in jsp

Reply:
Ooops thanks, copy/paste going wrong, fixing now.

Reply:
I still see an example of this in the changed files @georgel-pop-lr

Reply:
mmm I did a force push, should be fixed now, could you point out where it is?